src/index.html

```diff
@@ -2,10 +2,34 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+    <title>LeaderBoard</title>
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+    <div class="DocContainer">
+        <header></header>
+        <main class="appContainer">
+            <h1 class="titleHeader"><span class="header">Leader Board</span></h1>
+            <section class="appsection">
+                <section class="scoreDisplay">
+                    <article class="scoreHeader">
+                        <h2 class="scoreHeading">Recent Scores</h2>
+                        <button class="refreshScores">Refresh</button>
+                    </article>
+                    <ul class="scoresList">
+                    </ul>
+                </section>
+                <section class="scoreEntry">
+                    <article class="scoreEntryHeader">
+                        <h2 class="scoreEntryHeading">Add Your Score</h2>
+                    </article>
+                    <input type="text" required class="InputName" placeholder="Your Name">
+                    <input type="text" required class="Inputscore"placeholder="Your Score">
+                    <button class="submitScores">Submit</button>
+                </section>
+            </section>
+        </main>       
```

Comment:
- [ ] Lastly, per the project requirements, you need to ensure that you write plain HTML markup with minimum styling (just to make the layout), because you will implement styles in the following steps. 💯 
- Use this wireframe as a guide:
![image](https://github.com/gisachris/LeaderBoard/assets/102699958/749ea8e8-41ad-4a64-bc59-40095f3dd76f)

- But currently, your HTML markup does look like the given wireframe, so kindly implement this as well 😉 
- Your Design
<img width="743" alt="Screen Shot 2023-05-23 at 5 03 04 PM" src="https://github.com/gisachris/LeaderBoard/assets/102699958/57e63e5a-5102-4a66-9fcd-51f1acfb54d2">